app/src/main/java/com/ncnf/map/ui/MarkerInfoWindowManager.java

```diff
@@ -53,35 +51,40 @@ public void onClusterItemInfoWindowClick(NCNFMarker item) {
         if (item.isEvent()) {
 
             List<SocialObject> socialObjects = item.getEventList();
+            Fragment fragment;
 
             if (socialObjects.size() == 1) { //When the marker represents only one event
+                SocialObject e = item.getEventList().get(0);
```

Comment:
you can use a ternary if you want

Reply:
There are two commands in the if part, I don't think I can use a ternary here